modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -637,6 +637,60 @@ baseTest(
 		await waitForAlert(page);
 	}
 );
+baseTest(
+	'It ensures that translate side by side shows the duplicate fields',
+	{
+		tag: '@LPS-142169',
+	},
+	async ({apiHelpers, journalEditArticlePage, journalPage, page, site}) => {
+		const localizableFieldName = 'Text5678';
+		const structureName = 'Structure 1';
+
+		const dataDefinition = getDataStructureDefinition({
+			defaultLanguageId: 'en_US',
+			fields: [{name: localizableFieldName, repeatable: true}],
+			name: structureName,
+		});
+
+		await apiHelpers.dataEngine.createStructure(site.id, dataDefinition);
+
+		await journalEditArticlePage.goto({
+			siteUrl: site.friendlyUrlPath,
+			structureName,
+		});
+		const title = getRandomString();
+
+		await journalEditArticlePage.titleInput.fill(title);
```

Comment:
I think `fillAndClickOutside` will reduce possible flakiness, just like you did in https://github.com/liferay-content-management/liferay-portal/pull/5965/files